owlkettle/gtk.nim

```diff
@@ -340,6 +340,7 @@ proc g_signal_connect_data*(widget: pointer,
 proc g_type_from_name*(name: cstring): GType
 proc g_object_new*(typ: GType): pointer {.varargs.}
 proc g_object_ref*(obj: pointer)
+proc g_object_weak_ref*(obj: pointer, notify: proc (data, oldObjectPtr: pointer) {.cdecl.}, data: pointer)
```

Comment:
I feel tempted to leave this there just to have it as it shouldn't hurt, disagreement here?

Reply:
Yes, I would also leave it. It might be useful for debugging in the future.